src/main/scala/chisel3/tester/package.scala

```diff
@@ -39,11 +40,19 @@ package object tester {
         require(x.binaryPoint == value.binaryPoint, "binary point mismatch")
         pokeBits(x, value.litValue)
       }
-      case (x: Bundle, value: Bundle) => {
+      case (xx: Bundle, value: Bundle) => {
         // TODO: chisel needs to expose typeEquivalent
-        require(x.elements.keys == value.elements.keys)  // TODO: this discards type data =(
-        (x.elements zip value.elements) foreach { case ((_, x), (_, value)) =>
-          x.poke(value)
+        require(xx.elements.keys == value.elements.keys)  // TODO: this discards type data =(
+        (xx.elements zip value.elements) foreach { case ((fieldName, xxx), (_, value)) =>
+          try {
+            xxx.poke(value)
+          }
+          catch {
+            case _: NoSuchElementException =>
+              val bundleFieldName = x.toString.split("[(]").head + "." + fieldName
+              throw new LiteralValueException(
+                s"""Error: Maybe bundle field $bundleFieldName was not given a value during poke""")
```

Comment:
If you do use litOption, consider having this display both the underlying cause (incomplete bundle literal) and recommended resolution (consider filling in that field). Or we could support partial bundle literals, but that's another can of worms.

Reply:
I'm not quite sure how one would use litOption in this context. I think, for a minute, of proposing to relax the complete bundle requirement, but shied away from that.